composer.json

```diff
@@ -16,12 +16,14 @@
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "guzzlehttp/guzzle": "~6.0"
+        "guzzlehttp/guzzle": "~6.0",
+        "aws/aws-sdk-php": "~3.0"
```

Comment:
impossible to install this version using php 5.4, but above requirements say that it's >=5.4